client/src/components/WebAuthn/Verify.js

```diff
@@ -2,87 +2,198 @@
-    this.initAuth();
+    this.setState({
+      failure: false,
+    });
+  }
+
+  startAuth() {
+    const { publicKey, onCompleteVerification } = this.props;
+
+    performVerification(publicKey)
+      .then(verificationData => {
+        this.setState(
+          { success: true },
+          () => setTimeout(
+            () => { onCompleteVerification(verificationData); },
+            1000
+          )
+        );
+      })
+      .catch(() => this.setState({ failure: true }));
   }
 
   /**
-   * Render a description for this input
+   * Render instructions for registering with this method
    *
    * @return {HTMLElement}
    */
   renderDescription() {
     const { ss: { i18n } } = window;
+    const { method } = this.props;
+
+    const registerKeyT = i18n._t(
+      'MFAWebAuthnVerify.VERIFY',
+      fallbacks['MFAWebAuthnVerify.VERIFY']
+    );
+
+    // As this part of the message requires formatting, we render it using dangerouslySetInnerHTML
+    const instructions = i18n.inject(
+      i18n._t('MFAWebAuthnVerify.INSTRUCTION', fallbacks['MFAWebAuthnVerify.INSTRUCTION']),
+      { button: `<strong>${registerKeyT}</strong>` }
+    );
 
     return (
-      <p>
-        {i18n._t(
-          'MFAWebAuthnVerify.DESCRIPTION',
-          'Use your security key'
-        )}
-      </p>
+      <div className="mfa-verification-container__description">
+        <p>
+          {i18n._t('MFAWebAuthnVerify.DESCRIPTION', fallbacks['MFAWebAuthnVerify.DESCRIPTION'])}
+
+          <a
+            href={method.supportLink}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            {i18n._t('MFAWebAuthnVerify.HELP', fallbacks['MFAWebAuthnVerify.HELP'])}
+          </a>
+        </p>
+
+        {/* eslint-disable-next-line react/no-danger */}
+        <p dangerouslySetInnerHTML={{ __html: instructions }} />
+      </div>
     );
   }
 
+  /**
+   * Render the status of the registration, with relevant iconography
+   *
+   * @returns {HTMLElement}
+   */
+  renderStatus() {
+    const { ss: { i18n } } = window;
+    const { failure, success } = this.state;
 
-  render() {
+    if (success) {
+      return (
+        <div className="mfa-verification-container__status status-message--success">
+          <span className="status-message__icon"><CircleTick size="32px" /></span>
+          <span className="status-message__description">
+            {i18n._t('MFAWebAuthnVerify.SUCCESS', fallbacks['MFAWebAuthnVerify.SUCCESS'])}
+          </span>
+        </div>
+      );
+    }
+
+    if (failure) {
+        return (
+          <div className="mfa-verification-container__status status-message--failure">
+            <span className="status-message__icon"><CircleWarning size="32px" /></span>
+            <span className="status-message__description">
+              {i18n._t('MFAWebAuthnVerify.FAILURE', fallbacks['MFAWebAuthnVerify.FAILURE'])}
+            </span>
+          </div>
+        );
+    }
+
+    return (
```

Comment:
There's also an "error" state which was added recently in #33 - might be worth copying that through to here as well. You can force that state by setting `SudoModeService::check()` to return false _after_ you've loaded the MFA app but _before_ you've selected security key as a verification method